util/spell.py

```diff
@@ -0,0 +1,52 @@
+import re
+import kenlm
+from heapq import heapify
+from collections import Counter
+
+# Define beam with for alt sentence search
+BEAM_WIDTH = 16384
+
+# Load language model
+MODEL = kenlm.Model('./data/lm/lm.arpa')
+
+def words(text):
+    "List of words in text."
+    return re.findall(r'\w+', text.lower())
+
+# Calculate word statistics
+WORDS = Counter(words(open('./data/spell/words.txt').read()))
```

Comment:
On the file descriptor, good catch.

The Counter's just left over from Norvig's code. I'll switch to set.